appinventor/blocklyeditor/src/workspace_svg.js

```diff
@@ -568,6 +571,18 @@ Blockly.WorkspaceSvg.prototype.customContextMenu = function(menuOptions) {
   };
   menuOptions.splice(3, 0, exportOption);
 
+  //Show or hide workspace SVG elements backpack, zoom, and trashcan
+  var workspaceOption = {enabled: true};
+  workspaceOption.text = this.chromeHidden ? Blockly.Msg.SHOW : Blockly.Msg.HIDE;
+  var displayStyle = this.chromeHidden ? 'block' : 'none';
+  workspaceOption.callback= function() {
+    self.backpack_.svgGroup_.style.display=displayStyle;
+    self.trashcan.svgGroup_.style.display=displayStyle;
+    self.zoomControls_.svgGroup_.style.display=displayStyle;
```

Comment:
The one thing missing here is the warning indicator. If I hide, I would expect that to go away as well.